src/pages/login.js

```diff
@@ -1,10 +1,61 @@
-import React from "react";
-import Header from "../components/header";
+import React, { useState } from "react";
+import { Link } from 'react-router-dom'
+import styled from 'styled-components';
+import Form from "../components/form";
+import DefaultLayout from "../components/defaultLayout";
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+
+const StyledLink = styled(Link)`
+  margin-left: 5px;
+  text-decoration: none;
+  :hover{
+    text-decoration: underline;
+  }
+`
+const initialLoginState = {login: "", password: "" };
+
 
 const Login = () => {
 
+  const [formState, setFormState] = useState(initialLoginState);
+
+  const formHandlers = (event) => {
+    const { value, id} = event.target;
+    setFormState({...formState, [id]: event.type === "change" ? value : value.trim()});
+  }
+
   return(
-    <Header title="Login page." />
+    <DefaultLayout title="Login page">
+      <Form>
+        <Box mb="10px">
+          <TextField label="Login" id="login" value={formState.login} onChange={formHandlers} onBlur={formHandlers} fullWidth />
+        </Box>
+        <Box mb="10px">
+          <TextField label="Password" id="password" value={formState.password} onChange={formHandlers} onBlur={formHandlers} fullWidth />
+        </Box>
+
+        <Box mb="10px">
+          <Button variant="contained" fullWidth>Sign in</Button>
+        </Box>
+        <Box fullWidth sx={{
```

Comment:
Usually such components with a large number of styles are placed in a separate `components` file. It's necessary to leave the structure of the main component less burdened.

_**Rules - styles separately, logic separately.**_

You can also move the `StyledLink` component to the `components` file.

[Example](https://github.com/fs/itis-frontend-app/blob/master/src/components/entity/EntityCard/components.js) components file